src/main/java/no/nav/veilarbperson/client/pdl/PdlClientImpl.java

```diff
@@ -37,22 +38,39 @@
 
     private final String hentPersonQuery;
 
+    private final String hentPersonBolkQuery;
+
```

Comment:
Common har en client for integrasjoner mot PDL Det kan være greit å bruke den her også, så blir det litt mindre kode

Reply:
Common har generisk client class for å gjøre oppslag mot PDL. Der vi  har ikke mulighet til å tilpasse token i Authorization eller Nav-consumer-token felte i headeren i requesten per i dag og kanskje vi trenger ikke å gjøre det i fremtidet men PDL dok sier at det må sendes 2*systemtoken for å kjøre hentPersonBolk operasjonen. Da tenker vi å la den være som det er nå og koden kan endres i common etterhvert hvis det trengs.